src/imfill.jl

```diff
@@ -0,0 +1,39 @@
+"""
+ Image filling Algorithm
+
+ ```
+ filled_img = imfill(img, interval)
+ filled_img = imfill(img, interval, value)
+ filled_img = imfill(img, interval, value, connectivity)
+ ```
+
+ Connected components of an image is found using flood-fill algorithm and returns a copy of
+ the original image after filling objects that falls in the range of interval.
+
+
+ Parameters:
+
+  -  img            = Input image (Boolean array type)
+  -  interval       = objects of size in this range will be filled with False
```

Comment:
Add "(default value is `(0,64)`)"

Reply:
Thank you Tim, out of curiosity, where the value 64 comes from?

Reply:
About 3 lines after this :smile:. But since you mention it, does it really make sense to provide a default here? Or should we just require the user to tell us the desired size directly?

Reply:
The only default I can imagine is `(-Inf,Inf)` in which case we fill all the holes in the image. What do you think?

Reply:
@juliohm Woudn't taking the default range `(-Inf,Inf)` confuse the algorithm and will fill the background also? 
Consider this for an example: 
![fillholesinabinaryimageexample_02](https://user-images.githubusercontent.com/23431811/50725158-5c32c400-111f-11e9-8caf-4206541917fd.png)

Reply:
I think it's better to just force the user to provide it.

Reply:
I agree with @timholy , I think it is better to make it a required argument of the function. A positional one so that we would call it with:

```julia
imfill(bw, (a,b))
```

Reply:
Okay I will make the required changes.

Reply:
You could just say "invalid size interval", it should be clear :+1: